README.md

```diff
@@ -19,11 +19,48 @@ If bundler is not being used to manage dependencies, install the gem by executin
 
 ## Usage
 
-TODO: Write usage instructions here
+For one-off requests:
+
+```ruby
+require 'globus/client'
+
+# NOTE: The settings below live in the consumer, not in the gem.
+client = Globus::Client.configure(
+  client_id: Settings.globus.client_id,
+  client_secret: Settings.globus.client_secret,
+  uploads_directory: Settings.globus.uploads_directory,
+  transfer_endpoint_id: Settings.globus.transfer_endpoint_id
+)
+client.mkdir(user_id: 'mjgiarlo', work_id: 1234, work_version: 1)
+```
+
+You can also invoke methods directly on the client class, which is useful in a
+Rails application environment where you might initialize the client in an
+initializer and then invoke client methods in many other contexts where you want
+to be sure configuration has already occurred, e.g.:
+
+```ruby
+# config/initializers/globus_client.rb
+Globus::Client.configure(
+  client_id: Settings.globus.client_id,
+  client_secret: Settings.globus.client_secret,
+  uploads_directory: Settings.globus.uploads_directory,
+  transfer_endpoint_id: Settings.globus.transfer_endpoint_id
+)
+
+# app/services/my_globus_service.rb
+# ...
+def create_user_directory
+  Globus::Client.mkdir(user_id: 'mjgiarlo', work_id: 1234, work_version: 1)
+end
+# ...
```

Comment:
Writing up usage documentation really drove this refactoring as I thought about how we ought to be injecting configuration into the client and interacting with it (e.g., in H2).